react/src/components/ManageImageResourceLimitModal.tsx

```diff
@@ -0,0 +1,150 @@
+import { useSuspendedBackendaiClient } from '../hooks';
+import BAIModal, { BAIModalProps } from './BAIModal';
+import { useWebComponentInfo } from './DefaultProviders';
+import ImageResourceFormItem from './ImageResourceFormItem';
+import { ManageImageResourceLimitModalMutation } from './__generated__/ManageImageResourceLimitModalMutation.graphql';
+import { ResourceLimitInput } from './__generated__/ManageImageResourceModalMutation.graphql';
```

Comment:
This code will not be auto-generated because it does not fit the GQL auto-generated code rules.
```typescript
import {
  ManageImageResourceLimitModalMutation,
  ResourceLimitInput,
} from './__generated__/ManageImageResourceLimitModalMutation.graphql';
```

Reply:
It follows that rule. I have already renamed the component. Please check again.